codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -215,7 +215,7 @@ private void activateInternal(DefaultStream stream) {
     private void deactivateInternal(DefaultStream stream) {
         if (activeStreams.remove(stream)) {
```

Comment:
I think we have a naming disconnect between the `activate`/`deactivate` on the stream and `concurrent` on the endpoint.  For example the stream still has `numActiveStreams()` and `activeStreams()` which is logically the same thing as the endpoint's `numConcurrentStreams()` but with a different name. Can make this consistent?